### DISCUSSION TITLE
Is this app safe?

### DISCUSSION THREAD

**Discussion by BlueX5007** [morning4coffe-dev/storylines#19] [Q&A] [answered]
I don't see any reviews but it could be thoroughly due to it being in Preview.

**Comment by morning4coffe-dev** [comment, marked as answer]
Hello, thank you for your question. I can indeed confirm that the app is safe, it has been tested, and it passed all the certifications on Microsoft Store. May I ask where don't you see any reviews? On MS Store? It may be due to your region. But around the world we already have many reviews, I think it is currently around 40.

**Reply by BlueX5007** [reply]
Well, I don't see any reviews on MS Store, any screenshots of reviews would be nice and I'm planning to download it if it's free :)

**Reply by morning4coffe-dev** [reply]
Here you go, this is how our Ratings and reviews section on the Store looks in the United States. 
![image](https://user-images.githubusercontent.com/78549750/179057798-8a8a6522-0a69-4db3-9e48-b6d40a51d7b2.png)
And yes, the app is fully open-source and free.
